src/test/java/seedu/commando/logic/commands/AddCommandTest.java

```diff
@@ -261,4 +263,29 @@ public void execute_add_missingStartEndDates() throws IllegalValueException {
         assertEquals(Messages.MISSING_TODO_DATERANGE_END, result.getFeedback());
         assertFalse(wasToDoListChangedEventPosted(eventsCollector));
     }
+
+    @Test
+    public void execute_add_quotedTitle() throws IllegalValueException {
+        CommandResult result = logic.execute("add `event from today to tomorrow`");
+        assertFalse(result.hasError());
+        assertTrue(wasToDoListChangedEventPosted(eventsCollector));
+        assertTrue(ifToDoExists(logic,
+            new ToDoBuilder("event from today to tomorrow")
+                .build()));
+    }
+
+    @Test
+    public void execute_add_trailingTextAfterQuotedTitle() throws IllegalValueException {
+        CommandResult result = logic.execute("add `event from today to` tomorrow");
+        assertTrue(result.hasError());
+        assertFalse(wasToDoListChangedEventPosted(eventsCollector));
+    }
+
+    @Test
+    public void execute_add_emptyQuotedTitle() throws IllegalValueException {
+        CommandResult result = logic.execute("add `  ` from today to tomorrow");
```

Comment:
empty title should also include add '' +... which is an empty string besides only spaces